packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -266,6 +272,27 @@ export function getGlobalStyles( rawStyles, rawFeatures ) {
 		customValues
 	);
 
+	// Adds normalized PX values for each of the different keys
+	const dimensions = Dimensions.get( 'window' );
+	const fontSizes = {};
+	[ 'core', 'theme', 'user' ].forEach( ( key ) => {
+		if ( features?.typography?.fontSizes[ key ] ) {
+			fontSizes[ key ] = features?.typography?.fontSizes[ key ]?.map(
+				( fontSizeObject ) => {
+					fontSizeObject.sizePx = getPxFromCssUnit(
+						fontSizeObject.size,
+						{
+							width: dimensions.width,
+							height: dimensions.height,
+							fontSize: 16,
+						}
+					);
+					return fontSizeObject;
+				}
+			);
+		}
+	} );
+
```

Comment:
Could we move this out to its own function? So we avoid `getGlobalStyles` being too big. Wdyt?

Reply:
Good idea!